frontend/src/metabase/nav/components/AppBar/AppBarLarge.styled.tsx

```diff
@@ -44,6 +46,31 @@ export const AppBarLeftContainer = styled.div<AppBarLeftContainerProps>`
       opacity: ${props => (props.isNavBarEnabled ? 1 : 0)};
     }
   }
+
+  ${props =>
+    !props.isLogoVisible &&
+    css`
+      ${SidebarButton} {
+        opacity: 1;
+      }
+
+      ${SidebarIcon} {
+        color: ${color("text-medium")};
+
+        &:hover {
+          color: ${color("brand")};
+        }
+      }
+
+      padding-left: ${!props.isNavBarEnabled && "1rem"};
```

Comment:
When the `LogoLink` below is set to `display: none` it removes itself from rendering altogether so I used the same padding the logo uses.
- <img width="77" alt="Screenshot 2022-12-23 at 6 10 36 PM" src="https://user-images.githubusercontent.com/1937582/209388392-a57436b5-ce4d-4117-ae01-f226f6d99a91.png">
- <img width="232" alt="image" src="https://user-images.githubusercontent.com/1937582/209388403-78ae3ef9-e37c-44c6-a797-79de56b9b4f5.png">